api/desecapi/mail_backends.py

```diff
@@ -6,6 +6,7 @@
 from django.core.mail.backends.base import BaseEmailBackend
 from djcelery_email.utils import dict_to_email, email_to_dict
 
+from desecapi import metrics
```

Comment:
please add an extra blank line after this (so that there are 2 blank lines, see https://www.python.org/dev/peps/pep-0008/)